pom.xml

```diff
@@ -132,8 +132,9 @@
         <extensions>true</extensions>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Dependabot looked for `org.codehaus.groovy:maven-compiler-plugin` for some reason, that's why this wasn't caught.